fr/privacy.md

```diff
@@ -0,0 +1,20 @@
+# Politique de Confidentialité
+
+Cette page documente comment Krystara.io acquière, stocke et utilise les données collectées auprès des utilisateurs.
+
+## Sommaire
+
+1. [Cookies](#browser-cookies)
+2. [Analytique](#analytics)
+
+## Cookies

```

Comment:
Is this correct? Is the term cookies (referring to [HTTP cookies](https://en.wikipedia.org/wiki/HTTP_cookie)) the same between languages?

Reply:
Yes. And since we don't use it for the food item "browser" is implied.